tests_additional/real_exchanges/test_okex.py

```diff
@@ -0,0 +1,138 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import pytest
+
+from octobot_commons.enums import TimeFrames, PriceIndexes
+from octobot_trading.enums import ExchangeConstantsMarketStatusColumns as Ecmsc, \
+    ExchangeConstantsOrderBookInfoColumns as Ecobic, ExchangeConstantsOrderColumns as Ecoc, \
+    ExchangeConstantsTickersColumns as Ectc
+from tests_additional.real_exchanges.real_exchange_tester import RealExchangeTester
+# required to catch async loop context exceptions
+from tests import event_loop
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+class TestOkexRealExchangeTester(RealExchangeTester):
+    EXCHANGE_NAME = "okex"
+    SYMBOL = "BTC/USDT"
+
+    async def test_time_frames(self):
+        time_frames = await self.time_frames()
+        assert all(time_frame in time_frames for time_frame in (
+            TimeFrames.ONE_MINUTE.value,
+            TimeFrames.THREE_MINUTES.value,
+            TimeFrames.FIVE_MINUTES.value,
+            TimeFrames.FIFTEEN_MINUTES.value,
+            TimeFrames.THIRTY_MINUTES.value,
+            TimeFrames.ONE_HOUR.value,
+            TimeFrames.TWO_HOURS.value,
+            TimeFrames.FOUR_HOURS.value,
+            TimeFrames.SIX_HOURS.value,
+            TimeFrames.TWELVE_HOURS.value,
+            TimeFrames.ONE_DAY.value,
+            TimeFrames.ONE_WEEK.value,
+            TimeFrames.ONE_MONTH.value
+        ))
+
+    async def test_get_market_status(self):
+        market_status = await self.get_market_status()
+        assert market_status
+        assert market_status[Ecmsc.SYMBOL.value] == self.SYMBOL
+        assert market_status[Ecmsc.PRECISION.value]
+        assert market_status[Ecmsc.PRECISION.value][Ecmsc.PRECISION_AMOUNT.value] >= 1e-08
```

Comment:
when this is <1, we need to fix it (like https://github.com/Drakkar-Software/OctoBot-Tentacles/blob/master/Trading/Exchange/bitmax/bitmax_exchange.py#L58)

Reply:
Ok :+1: I didn't remember that